osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -131,5 +133,34 @@ public IEnumerable<SliderTick> Ticks
                 }
             }
         }
+        public IEnumerable<SliderBouncer> Bouncers
+        {
+            get
+            {
+                var length = Curve.Distance;
+                var bouncerDistance = Math.Min(BouncerDistance, length);
+                var repeatDuration = length / Velocity;
+
+                for (var repeat = 0; repeat < RepeatCount; repeat++)
+                {
+                    if (repeat > 0)

```

Comment:
Invert this if statement

Reply:
Actually might as well just start iterating at repeat = 1

Reply:
@Shawdooow did you miss this?